docs/data/charts/areas/areas.md

```diff
@@ -0,0 +1,15 @@
+---
+product: charts
+title: Charts - Areas
+---
+
+# Charts - Areas
+
+<p class="description">Chart areas can express qualities about data, such as highlights, and comparisons.</p>
+
+> ⚠️ This feature isn't implemented yet. It's coming.
+>
+> 👍 Upvote [issue #7928](https://github.com/mui/mui-x/issues/7928) if you want to see it land faster.
+>
+> 💬 To have a solution that solves your needs, leave a comments on the [same issue](https://github.com/mui/mui-x/issues/7928).
```

Comment:
```suggestion
> 💬 To have a solution that meets your needs, leave a comment on the [same issue](https://github.com/mui/mui-x/issues/7928).
```

```suggestion
> 💬 To have a solution that meets your needs, leave comments on the [same issue](https://github.com/mui/mui-x/issues/7928).
```

Or another proposal:

```suggestion
> 👍 Upvote [issue #7928](https://github.com/mui/mui-x/issues/7928) if you want to see it land faster or comment  it make sure the new feature meets your needs.

Reply:
I like the idea of adding images, but I don't know which one to display. We could use those from material design article but I did not found licence of those images

https://m2.material.io/design/communication/data-visualization.html#types